src/widget/city.c

```diff
@@ -137,7 +137,7 @@ static void build_start(const map_tile *tile)
 
 static void build_move(const map_tile *tile)
 {
-    if (!building_construction_in_progress() || !tile->grid_offset) {
+    if (!building_construction_in_progress()) {
         return;
     }
     building_construction_update(tile->x, tile->y, tile->grid_offset);
```

Comment:
Because of the way the red highlight works when moving the cursor out of the map. The new code allows this to happen:

![visual_feedback_2](https://user-images.githubusercontent.com/6518876/57599463-a35a1700-754e-11e9-8b14-b43761a78543.gif)

The old code would make the red highlight disappear when the cursor went outside the map, even though everything would still be deleted when the mouse button was released.